### PR TITLE
[stable/prometheus] Add support to pass parameters in the URL

### PR DESCRIPTION
**Pull request by halradaideh**
Pass parameters in the URL as ?key=val

No.
#### What this PR does / why we need it:
The title is enough

#### Special notes for your reviewer:
Thanks for all :P 
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by halradaideh** [comment]
any updates here ?
@mgoodness

**Comment by halradaideh** [comment]
@zanhsieh can you please help me here ?
`I1017 07:46:21.500] Error linting and installing charts: Error processing charts
I1017 07:46:21.501] ------------------------------------------------------------------------------------------------------------------------
I1017 07:46:21.501]  ✖︎ prometheus => (version: "9.1.3", path: "stable/prometheus") > Error waiting for process: exit status 1
I1017 07:46:21.501] ------------------------------------------------------------------------------------------------------------------------
I1017 07:46:22.076] 23f63226c7323936bda281261a7cef7b87452ca4b7bdbd84f939ab2f3e9e60d8
W1017 07:46:22.094] Traceback (most recent call last):
W1017 07:46:22.094]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W1017 07:46:22.094]     main(ARGS.env, ARGS.cmd + ARGS.args)
W1017 07:46:22.095]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W1017 07:46:22.095]     check(*cmd)
W1017 07:46:22.095]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W1017 07:46:22.095]     subprocess.check_call(cmd)
W1017 07:46:22.096]   File "/usr/lib/python2.7/subprocess.py", line 186, in check_call
W1017 07:46:22.096]     raise CalledProcessError(retcode, cmd)
W1017 07:46:22.096] subprocess.CalledProcessError: Command '('./test/e2e.sh',)' returned non-zero exit status 1
E1017 07:46:22.096] Command failed
I1017 07:46:22.096] process 639 exited with code 1 after 14.1m
E1017 07:46:22.097] FAIL: pull-charts-e2e`

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/17354/pull-charts-e2e/1184733754388647937/

**Comment by halradaideh** [comment]
@maorfr cen you please help here ?

**Comment by zanhsieh** [comment]
@halradaideh 
Some user reported they disable node-exporter would get e2e test passed. But I dunno it's a good approach.

**Comment by halradaideh** [comment]
@zanhsieh 

who will give me approval ? :D

**Comment by halradaideh** [comment]
disabling the node exporter solved the issue

I think it is a port conflict issue, owners of the tests should take a look here.

**Comment by zanhsieh** [comment]
@halradaideh
The chart owners. Unfortunately @gianrubio seems lost interest on helm charts. I suggest you find another helm member or owner of this chart. You might reach them in kubernetes slack channel.

**Comment by kpucynski** [comment]
There was also problem with alertmanager connection: ```1017 07:43:03.502] level=error ts=2019-10-17T07:33:10.669Z caller=notifier.go:528 component=notifier alertmanager=http://10.0.0.231:9093/api/v1/alerts count=0 msg="Error sending alert" err="Post http://10.0.0.231:9093/api/v1/alerts: dial tcp 10.0.0.231:9093: connect: connection refused"```

Node exporter was fixed few days ago: https://github.com/helm/charts/pull/18066

**Comment by halradaideh** [comment]
@kpucynski 
I have modified the commits
put there is no place in the README where the annotations are explained, i put a comment above the related section as the other annotations ..

File: stable/prometheus/values.yaml
Comment (kpucynski) [inline stable/prometheus/values.yaml]: I think that disabling node-exporter entirely by default is not an good option and will not be accepted by reviewers if you don't have good arguments for it.
Unfortunately failing tests are not an good argument ;)
Please check PR which I mentioned in the comment - you can try to move node exporter to different port in CI test config.

Comment (hradaideh) [reply stable/prometheus/values.yaml]: so change ports and apply the hack in this PR ?

Comment (kpucynski) [reply stable/prometheus/values.yaml]: Try to provide separate CI config with ports defined - it may be enough.

Comment (hradaideh) [reply stable/prometheus/values.yaml]: and how can i do that ?

Comment (kpucynski) [reply stable/prometheus/values.yaml]: you can check, for example, how prometheus-operator is configured: `ci/test-values.yaml`

Comment (hradaideh) [reply stable/prometheus/values.yaml]: so i should create a directory called CI with custom values file with different ports number for the node exporter ?

Comment (kpucynski) [reply stable/prometheus/values.yaml]: just check how it's done in other charts and prepare something similar

**Comment by zanhsieh** [comment]
@halradaideh 
Would you mind to resolve the file conflicting please? Thank you.
stable/prometheus/Chart.yaml

**Comment by halradaideh** [comment]
@zanhsieh i don't see a conflict, can you point it please ?

**Comment by halradaideh** [comment]
@zanhsieh  my bad, resolved

File: stable/prometheus/values.yaml
Comment (vsliouniaev) [inline stable/prometheus/values.yaml]: Please do not change this value

**Review by vsliouniaev — CHANGES_REQUESTED** [review]

File: stable/prometheus/values.yaml
Comment (hradaideh) [reply stable/prometheus/values.yaml]: i don't want to, put the test keeps failing

Comment (vsliouniaev) [reply stable/prometheus/values.yaml]: The tests have been fixed since

**Comment by vsliouniaev** [comment]
/hold
This PR has now been superceded by another rules update.
Please feel free to close it

**Comment by halradaideh** [comment]
@vsliouniaev what should i do now ?

**Comment by vsliouniaev** [comment]
If you still need some features from here you car reopen the PR or make a new one

**Pull request closed without merging**
